x-pack/legacy/plugins/monitoring/server/lib/cluster/get_clusters_from_request.js

```diff
@@ -181,22 +178,8 @@ export async function getClustersFromRequest(
   // add logstash data
   if (isInCodePath(codePaths, [CODE_PATH_LOGSTASH])) {
     const logstashes = await getLogstashForClusters(req, lsIndexPattern, clusters);
-    const pipelines = await getLogstashPipelineIds(req, lsIndexPattern, { clusterUuid }, size);
```

Comment:
I think the removal of this code is the real win here, however I don't know if we can safely remove it. There is a comment below:
>  withhold LS overview stats until pipeline metrics have at least one full bucket

That indicates there was an explicit reason for the code that is removed in this PR. Did you follow up with the logstash team to figure out if we can safely change this?

Reply:
Yeah, I saw that, but couldn't find a valid reason in blame/history since it was moved to x-pack directory not too long ago.

Just by looking at the code I think it was in place to prevent [sending zeros](https://github.com/elastic/kibana/pull/58205/files#diff-9120d0023b3c1458b40c27e53bdedef3) and instead just show nothing (if there are no buckets), since zeros could also mean valid values (and this might get misinterpreted). If that's the case, I don't think the benefits outweigh the cost here, since those zero's would  only show up at the initiation of monitoring and only for about 10 seconds

Reply:
Looks like it was in a PR done by @ycombinator and @justinkambic back in the x-pack repo

Reply:
@chrisronline Nice catch!

This was indeed a legitimate bug that was breaking the UI and causing errors: https://github.com/elastic/x-pack-kibana/issues/4517

Should be fixed now 👍